frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -9,18 +8,16 @@ import {
   RepoFileList,
 } from '../types';
 import { BaseService } from './base-service';
+import { coFetchJSON } from '@console/internal/co-fetch';
 
 export class BitbucketService extends BaseService {
-  private readonly client: Bitbucket;
-
   private readonly metadata: RepoMetadata;
 
+  private readonly baseURL = 'https://api.bitbucket.org/2.0';
+
   constructor(gitsource: GitSource) {
     super(gitsource);
     this.metadata = this.getRepoMetadata();
-    this.client = new Bitbucket();
-    const creds = this.getAuthProvider();
```

Comment:
We don't use auth right now but i thought we wanted to try when using a private repository?

Reply:
I created a new issue to track the feature enhancement that would utilize auth.
https://issues.redhat.com/browse/ODC-3797